csfieldguide/chapters/content/en/software-engineering/software-engineering.md

```diff
@@ -5,7 +5,7 @@
 # Challenges in teaching Software Engineering
 
 Software engineering is a challenging topic to teach for several reasons.
-One is that many, if not most, textbooks aren't written by practising software engineers, and often contain outdated ideas that are difficult for students to understand anyway (for example, the waterfall model is often taught near the start, and yet often doesn’t work in practice, and in fact was originally written to tell people how *not* to do software engineering).
+One reason is that many, if not most, textbooks aren't written by practising software engineers, and often contain outdated ideas that are difficult for students to understand. For example, the waterfall model is often taught near the start, and yet often doesn’t work in practice, and in fact was originally written to tell people how *not* to do software engineering.
```

Comment:
Regarding project style, we start new sentences on a new line &mdash; It makes it easier to understand when we revisit/compare content, and the single newline is replaced with a space when the document is viewed